langchain_postgres/chat_message_histories.py

```diff
@@ -320,6 +320,7 @@ def get_messages(self) -> List[BaseMessage]:
         with self._connection.cursor() as cursor:
             cursor.execute(query, {"session_id": self._session_id})
             items = [record[0] for record in cursor.fetchall()]
+        self._connection.commit()
```

Comment:
Does a transaction need to be committed here?  There's no mutation that's done in get_messages

Reply:
While doing some research I stumbled on [this Article]( https://lerner.co.il/2015/09/17/in-postgresql-as-in-life-dont-wait-too-long-to-commit/), which describes some implications of having not committed transactions.

Basically, it avoids `VACUUM` marking rows as eligible for reuse, as long as the connection is open.

That means, that if someone builds an application that is rarely restarted using this library, it can grow the size of the DB drastically.